movie-app/src/body/Searched.js

```diff
@@ -1,52 +1,104 @@
-import MovieCard from '../components/MovieCard'
-import MovieDetails from '../components/MovieDetails'
+import MovieCard from "../components/MovieCard";
+import MovieDetails from "../components/MovieDetails";
 
-const Searched = ({ totalResults, movies, setPage, page, toggleModal, selected, setSelected, modalOpen, setModalOpen, detailClick }) => {
+const Searched = ({
+  totalResults,
+  movies,
+  setPage,
+  page,
+  toggleModal,
+  selected,
+  setSelected,
+  modalOpen,
+  setModalOpen,
+  detailClick,
+}) => {
+  const moviesCheck = () => {
+    if (movies) {
+      return movies.map((movie) => {
+        return (
+          <MovieCard
+            toggleModal={toggleModal}
+            key={movie.imdbID}
+            movie={movie}
+            setSelected={setSelected}
+            detailClick={detailClick}
+          />
+        );
+      });
+    } else {
+      return <div>Loading...</div>;
+    }
+  };
 
-    const moviesCheck = () => {
-        if (movies) {
-            return movies.map((movie) => {
-                return <MovieCard
-                    toggleModal={toggleModal}
-                    key={movie.imdbID}
-                    movie={movie}
-                    setSelected={setSelected}
-                    detailClick={detailClick}
-                />;
-            });
-        } else {
-            return <div>Loading...</div>;
-        }
-    };
+  const scrollToTop = () => {
+    window.scrollTo({
+      top: 0,
+      behavior: "auto",
+    });
+  };
 
-    let movieRemainder = parseInt(totalResults) % 10 === 0 ? parseInt(totalResults) / 10 : Math.floor(parseInt(totalResults) / 10 + 1)
+  let movieRemainder =
+    parseInt(totalResults) % 10 === 0
+      ? parseInt(totalResults) / 10
+      : Math.floor(parseInt(totalResults) / 10 + 1);
```

Comment:
Another nice readability refactor.  It's generally a good idea to not have lines longer than 80 characters.